db/schema.rb

```diff
@@ -1198,8 +1197,8 @@
   end
 
   create_table "worldwide_services", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
-    t.string   "name",            null: false
-    t.integer  "service_type_id", null: false
+    t.string   "name",            default: "", null: false
```

Comment:
Do you know what this is? 😬

Reply:
Nope - seems to come up in staging and integration though so seems to be part of the db.